docs/spark/how-to-guides/hdinsight-notebook-installation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing .NET for Apache Spark on Azure HDInsight's Jupyter Notebooks
+description: Learn how to install .NET for Apache Spark on Azure HDInsight's Jupyter Notebooks
+ms.date: 02/20/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Installing .NET for Apache Spark on Azure HDInsight's Jupyter Notebooks
+
+This how-to provides the steps to install .NET for Apache Spark on Azure HDInsight's Jupyter Notebooks.
+
+## Background
+
+While we can deploy .NET for Apache Spark on HDI clusters through the command line/Azure portal (you can check out [how to deploy a .NET for Apache Spark application to Azure HDInsight](../tutorials/hdinsight-deployment.md) and [how to submit a .NET for Apache Spark job to Azure HDInsight](hdinsight-deploy-methods.md)), notebooks provide a more interactive and iterative experience.
+
+Azure HDI clusters come with Jupyter notebook, so let's look how to configure those Jupyter notebooks to run .NET for Apache Spark. In order to use .NET for Apache Spark within your Jupyter Notebooks, a C# REPL is needed to execute your C# code line-by-line and to preserve execution state when necessary. We have integrated [Try .NET](https://github.com/dotnet/try) as our official dotnet REPL.
+
+To provide the end-to-end notebook integration, we added a .NET for Apache Spark kernel to Jupyter kernel along with necessary modifications to [Apache Livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic) to HDI notebooks. Since these code changes have not yet been merged into the respective open source projects, you'll need to modify some components in your HDInsight Spark cluster to utilize .NET for Apache Spark in Jupyter notebooks.
```

Comment:
The first sentence is a bit confusing to me, and I think it may just be because it is so long and has several specific technologies listed. Perhaps we can break into two sentences? 

"... we added a .NET for Apache Spark kernel to the Jupyter kernel. We also added the necessary modifications..."

Also, I'm unclear based on the current structure of this sentence what's being added. Is Livy being modified/added with the kernel? Or is Livy being added/modified alongside sparkmagic in the HDI notebooks themselves?

Reply:
I have separated the long sentence to make it clear. So I don't think we need to tell user so much details about how we are making the modification which might confuse them. It's changes in both Livy and sparkmagic. And we also add Spark .NET to Jupyter kernel. Please let me know what you think.